packages/pocket-ic/CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The function `PocketIcBuilder::with_state_dir` to specify a directory in which the state of the PocketIC instance can be preserved across the PocketIC instance lifetime
   (that directory should be empty when specified as `state_dir` for the very first time).
 - The function `PocketIcBuilder::with_nonmainnet_features` to specify that non-mainnet features (e.g., best-effort responses) should be enabled for the PocketIC instance.
+- Support for canister HTTP outcalls: a function `PocketIc::get_canister_http` to retrieve pending canister HTTP outcalls
+  and a function `PocketIc::mock_canister_http_response` to mock a response for a pending canister HTTP outcall.
```

Comment:
I wanted to make it clear what their scope is.

> you'd call them like pic.get_canister_http(...)

Isn't this merely a shortcut rather than anything conceptual?